spec/index.bs

```diff
@@ -4686,21 +4692,31 @@ dictionary GPUShaderModuleDescriptor : GPUObjectDescriptorBase {
 source-map-v3 format.
 Source maps are optional, but serve as a standardized way to support dev-tool
 integration such as source-language debugging [[SourceMap]].
-WGSL names in source maps compare by their code points and
-do not follow [=UAX15 Equivalences=] for [[!UnicodeVersion14|Unicode Version 14.0.0]].
-
-Note: User agents should consider issuing developer-visible warnings in most or all
-situations where display of two names with different code point sequences
-look the same to the reader, such as ones same under canonical equivalence with
-[=UAX15 Normalization Forms|Normalization Form C (NFC)=] or characters that look similarly.
+WGSL names (identifiers) in source maps [=string/is|identical to=] each other are the same. As such, they do not compare
+through normalizations such as [=UAX15 Normalization Forms|Normalization Form C (NFC)=],
+[=UAX15 Normalization Forms|Normalization Form D (NFD)=], [=UAX15 Normalization Forms|Normalization Form KC (NFKC)=],
+[=UAX15 Normalization Forms|Normalization Form KD (NFKD)=],
+and further matching such as [=UTS46 Mapping=], and [=charmod-norm Matching=].
+
+Note: A user agent should issue developer-visible warnings when WGSL identifiers contain
+code points or code point sequences with homoglyphs since underlying code point sequences
+would cause the homographs to resolve to different WGSL identifiers despite looking
+the same to the reader. Examples of mappings to detect such display include
+normalizations such as [=UAX15 Normalization Forms|Normalization Form C (NFC)=],
+[=UAX15 Normalization Forms|Normalization Form D (NFD)=], [=UAX15 Normalization Forms|Normalization Form KC (NFKC)=],
+[=UAX15 Normalization Forms|Normalization Form KD (NFKD)=],
+and further matching such as [=UTS46 Mapping=], and [=charmod-norm Matching=].
```

Comment:
The above spec prose and associated seem to be copied verbatim in a few places. Could we establish a definition for something like "WGSL identifier comparison rules.", give this definition there, then link to it everywhere else with text along the lines of:

`WGSL names (identifiers) in source maps follow [=WGSL identifier comparison rules=].`

Reply:
@toji Thank you for the suggestion; that makes sense to me. If other editors agree to text wording changes, I will append a commit to form such and deduplicate in this PR!

Reply:
@toji I hope the latest commit's deduplication resolves this without breaking the existing opinion!